src/plugins/analysis-platform/plugins/sima-application-input/src/EditSimaApplicationInput.tsx

```diff
@@ -89,7 +91,7 @@ export const EditSimaApplicationInput = (props: IDmtUIPlugin) => {
     }
   }
 
-  if (loading || formData === null) return <Loading />
+  if (loading || document === null || formData === null) return <Loading />
```

Comment:
without checking `document === null`, typecsript compains further down . 

document === null and loading is kind of the same thing, so I am very open to other suggestions for how ot solve this.